pkg/apiserver/handlers/webhook/mutation_labels_test.go

```diff
@@ -239,3 +241,45 @@ func TestMutateLabels(t *testing.T) {
 		})
 	}
 }
+
+func TestHandleMutationLabels(t *testing.T) {
+	testCases := []struct {
+		name               string
+		contentType        string
+		requestBody        []byte
+		expectedStatusCode int
+	}{
+		{
+			name:               "empty request body",
```

Comment:
You may capitalize the first word of all test cases' names as others. Keep the format consistent.